topic-operator/src/main/java/io/strimzi/operator/topic/KafkaHandler.java

```diff
@@ -50,13 +50,16 @@
  */
 public class KafkaHandler {
     static final ReconciliationLogger LOGGER = ReconciliationLogger.create(KafkaHandler.class);
-
-    /** Default value for partitions and replicas. */
-    public static final int BROKER_DEFAULT = -1;
+    
     /** Kafka configuration for auto create topic. */
     public static final String AUTO_CREATE_TOPICS_ENABLE = "auto.create.topics.enable";
     /** Kafka configuration for min insync replicas. */
     public static final String MIN_INSYNC_REPLICAS = "min.insync.replicas";
+
+    /** Default value for partitions and replicas. */
+    public static final int DEFAULT_PARTITIONS_REPLICAS = -1;
```

Comment:
doesn't it make sense to have two separate default constants for partitions and replicas even if for now they are the same? Could they change in the future with different values?